examples/demos/rendering.js

```diff
@@ -20,6 +20,14 @@ function EventAgenda({ event }) {
   )
 }
 
+const customNow = () => {
```

Comment:
i don't think we need this addition to the example here?

Reply:
agreed! esp since there's a storyboard. Will drop.